packages/dev/core/src/Physics/v2/Plugins/havokPlugin.ts

```diff
@@ -288,7 +288,7 @@ export class HavokPlugin implements IPhysicsEnginePluginV2 {
     /**
      * We only have a single raycast in-flight right now
      */
-    private _queryCollector: bigint;
+    private _queryCollector;

```

Comment:
Why was the type removed here?

Reply:
queryCollector is not a `bigint` but a `[bigint]`. It's actually an opaque type that has no meaning outside Havok API calls.